### PR TITLE
Update most read rank for Persian and Pashto

### LINKED ISSUE #3233: Reduce width of double digit for pashto and persian
**Describe the bug**
As a result of this PR: https://github.com/bbc/psammead/pull/3201
Pashto and Persian have a non-optimal spacing between the rank value and the item text. 

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'the storybook application: http://localhost:8180/?path=/story/components-mostread-list--default'
2. Click on 'Pashto or Persian as a service'
3. See error

**Expected behaviour**
There should be less space between the rank value and item text. We adjust this by updating the min-width of the most read rank. This is currently affecting Pashto and Persian. There is no optimal width between the rank and the text atm but other services have spacing around 16px.

Should confirm with UX after you've fixed this bug.

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by thekp**
Resolves #3233

**Overall change:** _Update the width of most read's rank for Persian and Pashto._

**Code changes:**

- _Add a separate minWidth config for Persian and Pashto_
- _Updated `thinFontServices` to `mediumFontService` and added `smallFontService`._
- _Update snapshots_

**After:**
image
image
image

**Before:**
image
image
image
image

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-most-read/src/Rank/index.jsx [hunk 1]
```diff
@@ -132,9 +141,7 @@ const MultiColumnWrapper = styled(TwoColumnWrapper)`
   /* 5 columns of items at viewport 1280px and above */
   @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
     min-width: ${props =>
-      listHasDoubleDigits(props.numberOfItems)
-        ? isFiveOrTen(props)
-        : getRankMinWidth(props).group5};
+      listHasDoubleDigits(props.numberOfItems) ? isFiveOrTen(props) : 'auto'};
```
Comment (thekp) [inline packages/components/psammead-most-read/src/Rank/index.jsx:144]: I update this to `auto` as when there is no double-digit (i.e numbers 1-5) on the 5th group, the numbers will be always be aligned side by side and never on top of each other so there is no need to set a min-width for alignment. 

image

File: packages/components/psammead-most-read/src/utilities/rankMinWidth.js [hunk 2]
```diff
@@ -59,3 +57,22 @@ export const doubleDigitThin = {
...
 };
+
+export const singleDigitSmall = {
+  group0: GEL_SPACING_DBL,
+  group1: GEL_SPACING_DBL,
+  group2: GEL_SPACING_DBL,
+  group3: GEL_SPACING_TRPL,
+};
+
+export const doubleDigitSmall = {
```
Comment (thekp) [inline packages/components/psammead-most-read/src/utilities/rankMinWidth.js:68]: I opted to replicate the data structure above for this new group (`smallFontServices`) as I wouldn't have to change much here: `packages/components/psammead-most-read/src/Rank/index.jsx`

**Review by FK78 — APPROVED** [review]
👍

File: packages/components/psammead-most-read/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.0.4 | [PR#3291](https://github.com/bbc/psammead/pull/3291) Update rank min-width for persian and pashto |
```
Comment (EinsteinNjoroge) [inline packages/components/psammead-most-read/CHANGELOG.md]:
```suggestion
| 4.0.4 | [PR#3317](https://github.com/bbc/psammead/pull/3317) Update rank min-width for persian and pashto |
```

Comment (thekp) [reply packages/components/psammead-most-read/CHANGELOG.md]: Thanks 💯

**Review by EinsteinNjoroge — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Pull request merged**